src/mame/misc/rawthrillspc.cpp

```diff
@@ -119,12 +119,12 @@ ROM_END
 
 /*
  Doodle Jump Arcade, by default, uses a different PC than other Raw Thrills games:
-   -HP with an Asus M2N68-LA motherboard (socket AM3).
+   -HP with an Asus M2N68-LA motherboard (socket AM3, Fintek F8000 + Realtek RTL8201EL + Realtek ALC662 + nVidia NF-6100-430-N-A3 + ST L6740L).
```

Comment:
... which runs over nVidia MCP61P chipset, which I *think* it's still nForce based?

Reply:
Also the official M2N68-LA codename is "Narra 3"

Reply:
nVidia NF-6100-430-N-A3 is the integrated chip: GeForce 6100 + nForce 430

![IMG_7713](https://user-images.githubusercontent.com/5889052/220896273-26e46deb-9887-4c46-b40c-40388049971f.jpeg)

Reply:
What I'm trying to document with this PR are the chips and labels that can be seen or read on the actual PCBs, not the technical, marketing or commercial info about the components themselves.

Reply:
What I'm saying here is that yes, this makes this addition worth to stay in the same driver, but I'd like to have a note on top about being nForce chipset based (useful for later search).

Reply:
Ah! Ok!